.github/workflows/build-and-test.yml

```diff
@@ -47,3 +47,19 @@ jobs:
       - run: npm run build --if-present
       - run: npm start &
       - run: npm run test:integration
+
+  notify:
+    runs-on: ubuntu-latest
+    if: ${{ failure() }}
+    needs:
+      - test
+      - integrationtest
```

Comment:
Aside: If we have a `needs` context, can the alerter link to the failure?

Reply:
It does already. Not necessarily to the specific point of failure (there might be more than one), but to the Action run, from where it's fairly trivial to get to the point(s) of failure.

<img width="704" alt="Screenshot 2022-04-07 at 10 50 15" src="https://user-images.githubusercontent.com/117398/162172426-130d8f81-0a92-4638-a60e-42e3870678a5.png">

Reply:
gorrit